### PR TITLE
More Updates for AME-5308-WPA 3.3.4

### PR DESCRIPTION
**Pull request by austingene**
I noticed some files weren't updated in forked clone.

### CODE REVIEW

File: en/index.html [hunk 1]
```diff
@@ -185,6 +186,10 @@ <h1 class="title">ForgeRock Documentation</h1>
      </tr>
      <tr>
       <td class="name">OpenAM Policy Agents</td>
+      <td class="link version"><a href="index.html?product=openam-pa&version=3.3.4">3.3.4</a></td>
+     </tr>
+     <tr>
+      <td class="name">OpenAM Policy Agents</td>
```
Comment (markcraig) [inline en/index.html]: In this second line `OpenAM Policy Agents` should be a space, `&nbsp;`.

To see why I suggest this, view this file in your browser.

File: en/index.html [hunk 2]
```diff
@@ -309,6 +314,10 @@ <h1 class="title">OpenAM Access Management</h1>
       <td class="name">&nbsp;<!-- OpenAM Access Management --></td>
       <td class="link version"><a href="index.html?product=openam&version=10.0.0">10.0.0</a></td>
      </tr>
+     <tr>                                                                                  
+      <td class="name">OpenAM Policy Agents</td> 
+      <td class="link version"><a href="index.html?product=openam-pa&version=3.3.4">3.3.4</a></td>
+     </tr>
      <tr>
       <td class="name">OpenAM Policy Agents</td>
```
Comment (markcraig) [inline en/index.html]: Same thing here, use `&nbsp;`.

File: en/index.html [hunk 3]
```diff
@@ -697,6 +706,10 @@ <h1 class="title">OpenAM Policy Agents</h1>
     <tbody>
      <tr>
       <td class="name">OpenAM Policy Agents</td>
+      <td class="link"><a href="index.html?product=openam-pa&version=3.3.4">3.3.4</a></td>
+     </tr>
+     <tr>
+      <td class="name">OpenAM Policy Agents</td>
```
Comment (markcraig) [inline en/index.html]: And here

File: en/index.html (hunk 1 above)
Comment (austingene) [inline en/index.html]: I'll fix it. thx.

File: en/index.html (hunk 2 above)
Comment (austingene) [inline en/index.html]: Same here. thx.

**Pull request closed without merging**